lib/rubocop-rspec_rails.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'pathname'
+require 'yaml'
+
+require 'rubocop'
+require 'rubocop-rspec'
```

Comment:
What if we remove this line?

Reply:
The following error occurs:

```
 ❯ bundle exec rake
rake aborted!
NameError: uninitialized constant RuboCop::Cop::RSpec (NameError)

      class AvoidSetupHook < ::RuboCop::Cop::RSpec::Base
                                           ^^^^^^^
/ydah/tmp/rubocop-rspec/lib/rubocop/cop/rspec_rails/avoid_setup_hook.rb:19:in `<module:RSpecRails>'
/ydah/tmp/rubocop-rspec/lib/rubocop/cop/rspec_rails/avoid_setup_hook.rb:5:in `<module:Cop>'
/ydah/tmp/rubocop-rspec/lib/rubocop/cop/rspec_rails/avoid_setup_hook.rb:4:in `<module:RuboCop>'
/ydah/tmp/rubocop-rspec/lib/rubocop/cop/rspec_rails/avoid_setup_hook.rb:3:in `<top (required)>'
/ydah/tmp/rubocop-rspec/lib/rubocop/cop/rspec_rails_cops.rb:3:in `require_relative'
/ydah/tmp/rubocop-rspec/lib/rubocop/cop/rspec_rails_cops.rb:3:in `<top (required)>'
/ydah/tmp/rubocop-rspec/lib/rubocop-rspec_rails.rb:8:in `require_relative'
/ydah/tmp/rubocop-rspec/lib/rubocop-rspec_rails.rb:8:in `<top (required)>'
tasks/cops_documentation.rake:4:in `<top (required)>'
/ydah/tmp/rubocop-rspec/Rakefile:19:in `load'
/ydah/tmp/rubocop-rspec/Rakefile:19:in `block in <top (required)>'
/ydah/tmp/rubocop-rspec/Rakefile:19:in `each'
/ydah/tmp/rubocop-rspec/Rakefile:19:in `<top (required)>'
/ydah/.rbenv/versions/3.3.0/bin/bundle:25:in `load'
/ydah/.rbenv/versions/3.3.0/bin/bundle:25:in `<main>'
(See full trace by running task with --trace)
```

Reply:
Got it. We need to understand what those cops need.
Even though it’s impossible to use rspec-rails without rspec, I don’t think we should follow the same with our cops. I’d preferred if this could be used separately from rubocop-rspec, won’t rely on rubocop-rspec releases, won’t break when things get fixed/broken in rubocop-rapec etc. This would mean we can’t keep the same base. What is the best way to proceed?

Reply:
I tried to get rid of the dependency in https://github.com/ydah/rubocop-rspec_rails/pull/1/commits/e0cd330f08d3f9a015b770a07ccb9ba92248b4ad What do you think?